### PR TITLE
chore: Loader-setup

### PR DESCRIPTION
**Pull request by iamdevvalecha**
Setup loading bar #16

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -21,12 +29,29 @@ function MyApp({
...
+        <style jsx>{`
+          a {
+            margin: 0 10px 0 0;
+          }
+        `}</style>
+        <Link href="/">
+          <a></a>
+        </Link>
+      </nav>
+      <Component {...pageProps} />
```
Comment (iamareebjamal) [inline pages/_app.tsx:53]: Why are there 2 components?

Comment (iamdevvalecha) [reply pages/_app.tsx:53]: sorry i will resolve this

File: pages/_app.tsx [hunk 2]
```diff
@@ -21,12 +29,28 @@ function MyApp({
...
+      <ThemeProvider theme={theme}>
+        <StylesProvider injectFirst>
+          <CssBaseline />
+          <Component {...pageProps} err={err} />
+        </StylesProvider>
+      </ThemeProvider>
+      <Head>
+        {/* Import CSS for nprogress */}
+        <link rel="stylesheet" type="text/css" href="/nprogress.css" />
+      </Head>
```
Comment (iamareebjamal) [inline pages/_app.tsx:42]: The head element is below the page. How will the loader show above the page

Comment (iamdevvalecha) [reply pages/_app.tsx:42]: sorry i fix this

**Comment by iamdevvalecha** [comment]
closing this!

**Pull request closed without merging**